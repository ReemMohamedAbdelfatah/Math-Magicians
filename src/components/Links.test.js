import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Links from './Links';

describe('Link', () => {
  it('links component should match snapshot', () => {
    const homeLink = render(<BrowserRouter><Links /></BrowserRouter>);
    expect(homeLink).toMatchSnapshot();
  });
});
