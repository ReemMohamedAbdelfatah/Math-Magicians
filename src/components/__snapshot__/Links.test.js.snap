exports[`Test Link components render correctly 1`] = `null`;